src/main/java/com/redhat/jenkins/plugins/bayesian/Utils.java

```diff
@@ -45,10 +44,19 @@
         // Maven
         List<FilePath> mavenManifests = findManifestsFromList(workspace, knownMavenManifests);
         if (!mavenManifests.isEmpty()) {
-            // TODO: get all poms
-            FilePath pom = workspace.child("target/stackinfo/poms/pom.xml");
-            if (manifestExists(pom)) {
-                manifests.add(pom);
+            List<FilePath> childPomDirs = new ArrayList<FilePath>();
```

Comment:
We are allocating new `ArrayList` here, but then we immediately throw away this object on line number 53: https://github.com/fabric8-analytics/fabric8-analytics-jenkins-plugin/pull/3/files#diff-012a94a4fbf0120e16b99612bb4f803fR53

Note `mvn clean verify` actually fails the build on my machine because of this:
```
[INFO] --- findbugs-maven-plugin:3.0.4:check (findbugs) @ bayesian ---
[INFO] BugInstance size is 1
[INFO] Error size is 0
[INFO] Total bugs: 1
[INFO] Dead store to childPomDirs in com.redhat.jenkins.plugins.bayesian.Utils.findManifests(FilePath) [com.redhat.jenkins.plugins.bayesian.Utils] At Utils.java:[line 47] DLS_DEAD_LOCAL_STORE
[INFO] 
```